react-lists/React-app/src/App.tsx

```diff
@@ -9,7 +9,9 @@ const pokedex = [
 ];
 
 export default function App() {
-  const listItems = pokedex.map((pokemon) => <li>{pokemon.name}</li>);
+  const listItems = pokedex.map((pokemon) => (
+    <li key={pokemon.name}>{pokemon.name}</li>
```

Comment:
The keys all need to be unique. Usually you would use an ID of some type. In this case, `pokemon.number` is guaranteed to be unique.